cli/azd/pkg/commands/pipeline/pipeline_manager.go

```diff
@@ -273,8 +287,17 @@ func (manager *PipelineManager) Configure(ctx context.Context) error {
 
 	if doPush {
 		err = manager.pushGitRepo(ctx, currentBranch)
+		if err == nil {
```

Comment:
Should we be early exiting with `err` if `pushGitRepo` returns `err`?

Reply:
done